capirca/lib/naming.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, naming_dir=None, naming_file=None, naming_type=None):
     self.unseen_networks = {}
     self.port_re = re.compile(r'(^\d+-\d+|^\d+)\/\w+$|^[\w\d-]+$',
                               re.IGNORECASE|re.DOTALL)
+    self.token_re = re.compile(r'(^[-_A-Z0-9]+$)')
```

Comment:
Do we disallow uppercase on purpose? (I don't see a re.IGNORECASE here, or we could just do A-z instead of A-Z)

Reply:
Friendly ping :)

Reply:
Whoops! I thought I replied to this. I'm updating this PR today